libXfont must be statically linked due to poor weak symbol support on PE/COFF.
We handle this with a hack in the .cygport, so the rest of the libraries
can be linked dynamically.

This patch is NOT suitable for pushing upstream.

--- origsrc/xorg-server-1.7.0.901/hw/xwin/Makefile.am	2009-08-10 23:00:25.000000000 -0500
+++ src/xorg-server-1.7.0.901/hw/xwin/Makefile.am	2009-10-13 22:09:53.845230000 -0500
@@ -153,7 +153,7 @@ XWin_LDADD = $(MULTIWINDOWEXTWM_LIBS) $(
 .rc.o:
 	$(WINDRES) --use-temp-file -i $< --input-format=rc -o $@ -O coff -DPROJECT_NAME=\"$(VENDOR_NAME_SHORT)\"
 
-XWin_LDFLAGS = -mwindows -static
+XWin_LDFLAGS = -mwindows
 
 winprefsyacc.h: winprefsyacc.c
 winprefslex.c: winprefslex.l winprefsyacc.c winprefsyacc.h
